### DISCUSSION TITLE
FBW A320neo flightplan

### DISCUSSION THREAD

**Discussion by Splka1** [flybywiresim/aircraft#7178] [Q&A]
After normal take-off and a fully configured MCDU, activating the autopilot (set to NAV) will immediatley return the plane to the departure airport and start the loaded flightplan from there. Once 'back' to the airpot it start following the flightplan without any issues.
Any ideas? All latest updates in MSFS2020 and FBW A32x are installed, no other add-ons.

**Comment by tracernz** [comment]
What is the active/to waypoint prior to departure (top right of ND, or highlighted white on the MCDU F-PLN page)?
